apps/user_profile/forms.py

```diff
@@ -0,0 +1,119 @@
+from django.forms import CharField, Form, PasswordInput, IntegerField
+from django import forms
+from django.views.generic import FormView
+
+from apps.user_profile.models import Child
+
+
+class RegisterChildForm(forms.ModelForm):
+        class Meta:
+            model = Child
+            fields = ('first_name', 'last_name', 'surname', 'dob', 'sex', 'city', 'street', 'house', 'flat',
```

Comment:
so many fields in list)))
refactor that like this https://docs.djangoproject.com/en/2.0/topics/forms/modelforms/#selecting-the-fields-to-use
You should to add 
`fields = '__all__'`
`exclude = ['some_fields']`